staging/src/k8s.io/apimachinery/pkg/api/meta/priority.go

```diff
@@ -153,7 +153,7 @@ func kindMatches(pattern schema.GroupVersionKind, kind schema.GroupVersionKind)
 }
 
 func (m PriorityRESTMapper) RESTMapping(gk schema.GroupKind, versions ...string) (mapping *RESTMapping, err error) {
-	mappings, err := m.Delegate.RESTMappings(gk)
+	mappings, err := m.Delegate.RESTMappings(gk, versions...)
```

Comment:
@deads2k does passing these through look correct to you? is the underlying delegate intended to know about the user versions, or does PriorityRESTMapper take responsibility for making use of those versions?

Reply:
> @deads2k does passing these through look correct to you? is the underlying delegate intended to know about the user versions, or does PriorityRESTMapper take responsibility for making use of those versions?

They should be passed through.  With our current chain, I think the end result came out the same, but each constituent should get its shot at the information.

Reply:
On closer inspection, this will also fix a bug with kubectl trying to submit a version of a kind the server doesn't have to a version the server does have

for example:
```
$ more role.yaml 
apiVersion: rbac.authorization.k8s.io/v2
kind: ClusterRole
...

kubectl create -f role.yaml --v=6
...
POST https://localhost:6443/apis/rbac.authorization.k8s.io/v1
...
Error from server (BadRequest): error when replacing "role.yaml": ClusterRole in version "v2" cannot be handled as a ClusterRole: no kind "ClusterRole" is registered for version "rbac.authorization.k8s.io/v2"
```

the client should not have attempted to send the v2 object to the v1 endpoint... it should have failed when looking up the rest mapping